### ISSUE TITLE
Prep /me for website transition

### ISSUE THREAD

**Issue by xmatthewx** [mozilla/id.webmaker.org#339]
Users will need to access their existing makes when we launch the new webmaker.org site. Currently, they do this at https://webmaker.org/me. 

I'd guess that the easiest solution is to redirect https://webmaker.org/me to old.webmaker.org/me. This a successful and uncomplicated archive with functioning login. 

@jbuck - what do you recommend?

**Comment by xmatthewx** [comment]
Is there an export tool or URL as well? Any concerns with that?

**Comment by xmatthewx** [comment]
cc @simonwex

**Comment by jbuck** [comment]
Yes, I think doing the redirect from webmaker.org/me to old.webmaker.org/me is fine. We'll also need to change the config for `make-valet` (another service hosting makes.org) to set `WEBMAKERORG=https://old.webmaker.org` (currently `https://webmaker.org`)

There's no export tool for makes. What do you mean by URL though?

**Comment by xmatthewx** [comment]
I thought there might be a download-all-my-stuff url, but I haven't followed the conversations on popcorn etc. I don't know anything about /me or make-valet, so I'll let you steer this ship through the fog. :foggy: :ship:
